content/post/094-gossipsub-v1.1.md

```diff
@@ -0,0 +1,79 @@
+
+**4) [Prune Peer eXchange](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#prune-backoff-and-peer-exchange)**: Prune Peer exchange gives a node a set of recommended peers to connect when it gets dropped from another peer’s mesh, helping reduce the dependency on ambient peer discovery mechanisms.
+
+**5) [Extended Message Validators](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#extended-validators)**: You can now add your custom message validator so that you can instrument Gossipsub to recognize (accept/reject) which messages are valid/invalid within the context of your application.
+
+**6) [Outbound Mesh Quotas](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#outbound-mesh-quotas)**: In v1.0 mesh peers are randomly selected, without any weight given to the direction of the connection. In contrast,  v1.1 implements outbound connection quotas, so that a peer tries to always maintain a number of outbound connections in the mesh.
+
+The blend of these security measures, together with the properties of the “gossiping mesh” check the two most important boxes for message propagation protocols in permissionless networks, namely: i) fast message propagation, ii) security against attacks that attempt to degrade the quality of the service provided by the network.
+
+All of these upgrades can be consulted in the [Gossipsub v1.1 specification](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/README.md), which if you ask us, it is a joy to read!
+
+Additionally, with the new release, the [reference implementation of Gossipsub](https://github.com/libp2p/go-libp2p-pubsub) got additional documentation and test coverage.
+ 
+## 🏎 Give Gossipsub v1.1 a test ride
+
+### Chat application example
+
+We’ve prepared a brand new example for you to give the latest Gossipsub a try. In this example, you get to experiment with a chat application powered by libp2p. It will enable you to join different channels by subscribing to PubSub topics and seeing other peers joining the room as they subscribe to the topic.
+
+<p align="center">
+  <img width="60%" src="https://raw.githubusercontent.com/libp2p/go-libp2p-examples/master/pubsub/chat/chat-example.gif">
+</p>
+
+You can find the code for this example at https://github.com/libp2p/go-libp2p-examples/tree/master/pubsub/chat
+
+### Test Gossipsub’s performance with Testground
+
+We are also releasing a brand new libp2p Test Plan that uses [Testground](https://github.com/testground/testground) to benchmark Gossipsub v1.1.
+
+<p align="center">
+  <img width="60%" src="/img/gossipsubv1.1-jupyter.gif">
```

Comment:
```suggestion
  <img src="/img/gossipsubv1.1-jupyter.gif" style="max-width:600px">
```

Reply:
600px might not be the right size for all of these but play around and see what looks best 👀